packages/expo-location/android/src/main/java/expo/modules/location/LocationHelpers.java

```diff
@@ -119,16 +119,15 @@ static Bundle headingToBundle(double trueNorth, double magneticNorth, int accura
     return heading;
   }
 
-  public static LocationRequest prepareLocationRequest(Map<String, Object> options) {
+  public static LocationRequest.Builder prepareLocationRequest(Map<String, Object> options) {
     LocationParams locationParams = LocationHelpers.mapOptionsToLocationParams(options);
     int accuracy = LocationHelpers.getAccuracyFromOptions(options);
 
-    return new LocationRequest()
-        .setFastestInterval(locationParams.getInterval())
-        .setInterval(locationParams.getInterval())
-        .setMaxWaitTime(locationParams.getInterval())
-        .setSmallestDisplacement(locationParams.getDistance())
-        .setPriority(mapAccuracyToPriority(accuracy));
+    return new LocationRequest.Builder(locationParams.getInterval())
```

Comment:
`LocationRequest.Builder` cannot be used on devices with API below 31 - it's unavailable. We must wrap that with an API check if you want to use it.

Reply:
LocationRequest itself was added in 31. It would have never worked below 31 in that case? Wouldn't we get a warning if it required a check? There's compact classes available so I'll use that

Reply:
I also ran the tests on a device on API 29.

Reply:
It works. There are two packages. One comes from `android` the other from `com.google.android.gms`. I imported the Android one to check and you do get the warning in that case but we were never using those

Reply:
Ok, I'll merge it ;) Thanks for the clarification.